HollowEngine/HollowEngine/src/Hollow/Managers/ScriptingManager.cpp

```diff
@@ -40,6 +70,7 @@ namespace Hollow
 			"__sub", [](const glm::vec2& l, const glm::vec2& r) { return glm::vec2(l.x - r.x, l.y - r.y); }
 			);
 
+		// COMPONENTS
 		lua.new_usertype<Body>("RigidBody",
```

Comment:
Whenever we want a new function in a script should we just add the lua binding in this file?

Reply:
Yes